scripts/dev-multiuser.sh

```
@@ -0,0 +1,227 @@
+  sleep 1
+  exit 0
+}
+trap cleanup INT TERM EXIT
+
+# --- 5. Start client (vite, as developer, from the worktree) ---------------
+# Vite runs against the developer's worktree (NOT $TARGET_HOME), so client
+# edits propagate through HMR as usual. The vite dev server proxies API
+# requests to localhost:$PORT (server below), so the client/server boundary
+# is identical to production from the browser's perspective.
+echo "[4/5] Starting vite client (as $CURRENT_USER, from worktree) on port $CLIENT_PORT..."
+(cd "$REPO_ROOT/packages/client" && CLIENT_PORT="$CLIENT_PORT" bun run dev --port "$CLIENT_PORT") 2>&1 \
```

Shellcheck (0.11.0)

[warning] This assignment is only seen by the forked process.

(SC2097)